lib/rubocop-rspec_rails.rb

```diff
@@ -16,3 +16,22 @@
 
 project_root = File.join(__dir__, '..')
 RuboCop::ConfigLoader.inject_defaults!(project_root)
+
+# FIXME: This is a workaround for the following issue:
+# https://github.com/rubocop/rubocop-rspec_rails/issues/8
+module RuboCop
+  module Cop
+    class AmbiguousCopName # rubocop:disable Style/Documentation
+      prepend(Module.new do
+        def qualified_cop_name(name, path, warn: true)
```

Comment:
memo) There are two possibilities: Rails/HttpStatus or RSpecRails/HttpStatus, which causes an AmbiguousCopName Error. So, as a work around, if name is' RSpec/Rails/HttpStatus', it is rigged so that an AmbiguousCopName Error does not occur.

```
Error: Ambiguous cop name `RSpec/Rails/HttpStatus` used in /ydah/sandbox_capybara_with_rspec/.rubocop.yml needs department qualifier. Did you mean Rails/HttpStatus or RSpecRails/HttpStatus?
```

Refs: https://github.com/rubocop/rubocop/blob/97814aa1628789c338e66da7c201fe46dd6aa469/lib/rubocop/cop/registry.rb#L133-L145